api/tests/Feature/PoolCandidateUpdateTest.php

```diff
@@ -1034,7 +1033,11 @@ public function testManualStatusUpdatesTimestamps($status, $timestamp, $addition
         $unChangeData = $noChangeResponse['data']['updatePoolCandidateStatus'];
         $unchanged = new Carbon($unChangeData[$camelTimestamp]);
 
-        $this->assertEquals($new->timestamp, $unchanged->timestamp);
+        // Same year, month, day, hour, minute (close enough!)
+        $this->assertTrue($new->isSameDay($unchanged));
+        $this->assertTrue($new->isSameHour($unchanged));
+        $this->assertTrue($new->isSameHour($unchanged));
```

Comment:
Duplicated

Reply:
Removed in [remove duplicate line](https://github.com/GCTC-NTGC/gc-digital-talent/pull/12748/commits/dcb9c533ba0504be2adff0d6044ed07fa12de4ac)